core-bundle/src/EventListener/RequestTokenListener.php

```diff
@@ -93,7 +95,13 @@ public function __invoke(RequestEvent $event): void
             }
         }
 
-        $token = new CsrfToken($this->csrfTokenName, $this->getTokenFromRequest($request));
+        $value = $this->getTokenFromRequest($request);
+
+        if (Validator::isInsecurePath($value)) {
+            throw new BadRequestHttpException('Invalid CSRF token');
+        }
```

Comment:
Why is the CSRF token considered a path here?
Wouldn’t this error be checked by `isTokenValid()` from below and trigger the `InvalidRequestTokenException` anyways?

Reply:
The request had `../1` for `REQUEST_TOKEN`, which ultimately led to a `A non-numeric value encountered` warning.

Reply:
Wouldn’t `foo/1` then also lead to `A non-numeric value encountered`?
Should I investigate where the issue occurs and throw a proper `InvalidArgumentException` there?

Reply:
Nice. 😎

Reply:
I have reverted the changes in c8995f1152e9e9b84ec3a0c9259427cfa0316c27.